frontend/src/pages/RecipientRecord/pages/Monitoring/index.js

```diff
@@ -2,13 +2,14 @@ import React, {
   useEffect, useState, useMemo, useContext,
 } from 'react';
 import ReactRouterPropTypes from 'react-router-prop-types';
+import useDeepCompareEffect from 'use-deep-compare-effect';
 import { useHistory } from 'react-router';
 import Container from '../../../../components/Container';
 import Tabs from '../../../../components/Tabs';
 import { getTtaByCitation, getTtaByReview } from '../../../../fetchers/monitoring';
 import ReviewCards from './components/ReviewCards';
 import CitationCards from './components/CitationCards';
-import { ROUTES } from '../../../../Constants';
+// import { ROUTES } from '../../../../Constants';
```

Comment:
Need to remove commented out?

Reply:
oh shoot I need to handle the error in there, good catch